File: 2042-Check if Numbers Are Ascending in a Sentence-检查句子中的数字是否递增/2042-Check if Numbers Are Ascending in a Sentence-检查句子中的数字是否递增.java
class Solution {
    public boolean areNumbersAscending(String s) {
        String[] tokens = s.split(" ");
    	int last = -1;
    	for(int i = 0; i < tokens.length; i++) {
    		try {
    			int num = Integer.parseInt(tokens[i]);
        		if(last >= num) {
        			return false;
        		}
        		last = num;
			} catch (Exception e) {
				// TODO: handle exception
			}
    	}
    	return true;
    }
}

File: 2042-Check if Numbers Are Ascending in a Sentence-检查句子中的数字是否递增/2042-Check if Numbers Are Ascending in a Sentence-检查句子中的数字是否递增.js
/**
 * @param {string} s
 * @return {boolean}
 */
var areNumbersAscending = function(s) {
    let numTokens = s.match(/\d+/g)
    let last = numTokens[0]
    let len = numTokens.length
    for(let i = 1; i < len; i++){
        if(Number.parseInt(last) >= Number.parseInt(numTokens[i])){
            return false
        }
        last = numTokens[i]
    }
    return true
};
